Formula/re2.rb

```diff
@@ -1,9 +1,9 @@
 class Re2 < Formula
   desc "Alternative to backtracking PCRE-style regular expression engines"
   homepage "https://github.com/google/re2"
-  url "https://github.com/google/re2/archive/2020-07-06.tar.gz"
-  version "20200706"
-  sha256 "2e9489a31ae007c81e90e8ec8a15d62d58a9c18d4fd1603f6441ef248556b41f"
+  url "https://github.com/google/re2/archive/2020-08-01.tar.gz"
+  version "20200801"
```

Comment:
maybe we should remove this? livecheck cannot capture the upgrade.

```
$ brew livecheck re2
re2 : 20200706 ==> 2020-08-01

$ brew livecheck -n re2
No newer upstream versions.
```

Reply:
I agree that it's probably unnecessary. Writing a livecheck rule would be pretty easy too but I don't really see the point of formatting the `version` like this.

I don't mind if merging as-is and fixing later or if you want to stop this PR for now.